diff --git a/experimental/clashapi/connections.go b/experimental/clashapi/connections.go
index 2cf0a71..1df0f18 100644
--- a/experimental/clashapi/connections.go
+++ b/experimental/clashapi/connections.go
@@ -94,6 +94,7 @@ func (c connectionObject) MarshalJSON() ([]byte, error) {
 			"host":            domain,
 			"dnsMode":         "normal",
 			"processPath":     processPath,
+			"user":            c.Metadata.User,
 		},
 		"upload":      c.Upload.Load(),
 		"download":    c.Download.Load(),
